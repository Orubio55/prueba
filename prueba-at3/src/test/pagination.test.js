import { render } from "@testing-library/react";
import React from "react";

import Pagination from "components/pagination";

describe("Pagination", () => {
  it("should render successfully", () => {
    const { baseElement } = render(
      <Pagination data={[1, 2, 3]} countPage={2} changeList={() => null} />
    );
    expect(baseElement).toBeTruthy();
  });
  it("should match snapshot", () => {
    const { baseElement } = render(
      <Pagination data={[1, 2, 3]} countPage={2} changeList={() => null} />
    );
    expect(baseElement).toMatchSnapshot();
  });
});
