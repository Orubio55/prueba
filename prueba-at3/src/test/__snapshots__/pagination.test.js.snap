// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination should match snapshot 1`] = `
<body>
  <div>
    <div
      class="paginationBox"
    >
      <span
        class="pagination-element active "
      >
        1
      </span>
      <span
        class="pagination-element  "
      >
        2
      </span>
      <span
        class="pagination-element"
      >
        Â»
      </span>
    </div>
  </div>
</body>
`;
